### PR TITLE
removed landscape mode

### PR DESCRIPTION
**Pull request by fluidvanadium**

### CODE REVIEW

**Comment by juanky201271** [comment]
@fluidvanadium 

you need to remove this as well, in the Header:
https://github.com/fluidvanadium/zingo-mobile/blob/424e5849d576c585867a65463e4ed8c89151e32c/components/Header/Header.tsx#L110

I'm not testing this in local, I'm sure you tested in your local repo... :D

**Comment by fluidvanadium** [comment]
took care of that tricky lil' bit there

i'm testing using some scripts that are in another PR

File: components/Header/Header.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ const Header: React.FunctionComponent<HeaderProps> = ({
           </View>
         )}
 
-        {dimensions.orientation === 'landscape' && (
+        {(
           <View style={{ width: '100%', height: 1, backgroundColor: colors.primary }} />
         )}
```
Comment (juanky201271) [inline components/Header/Header.tsx]: you have to remove those 3 lines, because in `portrait` mode you don't need this horizontal line.

**Review by juanky201271 — CHANGES_REQUESTED** [review]

File: components/Header/Header.tsx (hunk 1 above)
Comment (fluidvanadium) [reply components/Header/Header.tsx]: oh oops! i didn't actually see that line, but yeah i shouldnt have added it. double extra fixed this time

File: components/Header/Header.tsx [hunk 2]
```diff
@@ -181,9 +177,6 @@ const Header: React.FunctionComponent<HeaderProps> = ({
         <Text style={{ fontSize: 8, color: colors.border }}>
           {'(' + dimensions.width + 'x' + dimensions.height + ')-' + dimensions.scale}
         </Text>
-        <Text style={{ fontSize: 7, color: colors.border }}>
-          {dimensions.orientation === 'landscape' ? translate('info.landscape') : translate('info.portrait')}
-        </Text>
```
Comment (juanky201271) [inline components/Header/Header.tsx:186]: ok, we can rid of it as well... this could make sense in Test, but now, I don't think so.

**Review by juanky201271 — DISMISSED** [review]

**Review by juanky201271 — DISMISSED** [review]
looks good.

**Review by juanky201271 — APPROVED** [review]
nice.

**Comment by zancas** [comment]
Let's use rebase instead of merge to update the feature branch to contain the latest dev code.

**Comment by zancas** [comment]
Superceded by #213

**Pull request closed without merging**